vertical-pod-autoscaler/docs/components.md

```diff
@@ -7,12 +7,15 @@
   - [Recommender](#recommender)
     - [Running](#running-the-recommender)
     - [Implementation](#implementation-of-the-recommender)
+    - [Flags](flags.md#what-are-the-parameters-to-vpa-recommender)
   - [Updater](#updater)
     - [Current implementation](#current-implementation)
     - [Missing Parts](#missing-parts)
+    - [Flags](flags.md#what-are-the-parameters-to-vpa-updater)
   - [Admission Controller](#admission-controller)
     - [Running](#running-the-admission-controller)
     - [Implementation](#implementation-of-the-admission-controller)
+    - [Flags](flags.md#what-are-the-parameters-to-vpa-admission-controller)
```

Comment:
My assumption is that the contents is a table of contents, and since the flags live in another file, this may be confusing.
What if a link was made under `Introduction` to link to the flags page?

Reply:
Hmm, that sounds reasonable.

Reply:
added in [037b377](https://github.com/kubernetes/autoscaler/pull/7618/commits/037b3773550e96c42404c0777a41e99b669c8eb6)